src/Tempest/Console/src/HasConsole.php

```diff
@@ -5,12 +5,12 @@
 namespace Tempest\Console;
 
 use Closure;
+use Tempest\Container\Inject;
 
 trait HasConsole
 {
-    public function __construct(private readonly Console $console)
-    {
-    }
+    #[Inject]
+    private readonly Console $console; // @phpstan-ignore-line
```

Comment:
Why do we need to ignore something from phpstan here ?
It will be always the case in all other implementations ?

Reply:
The ignore-line tag actually doesn't work, since it's being used via a trait… IDK yet what the best solution is